scriptmodules/emulators/mupen64plus.sh

```diff
@@ -245,7 +245,8 @@ function configure_mupen64plus() {
 
         # Disable gles2n64 autores feature and use dispmanx upscaling
         iniConfig " = " "" "$md_conf_root/n64/gles2n64.conf"
-        iniSet "auto resolution" "0"
+        # Note: does not like spaces around the '='
+        iniSet "auto resolution=0"
```

Comment:
I don't really like this change. It's an ugly hack :)

You should instead do (notice I removed the spaces in the first parameter):

    iniConfig "=" "" "$md_conf_root/n64/gles2n64.conf"

To make every `iniSet` call to not use spaces around the `=` as it is needed.
Then you leave the `iniSet` call as it was. This is the way the `inifuncs` is meant to be used :)

Reply:
oh, yes definitely that is better! i didn't notice it had that ability. will change

Reply:
done! now doing it the proper way

Reply:
Looks good! I will assume you tested all so will approve the change :)